src/_panel.scss

```diff
@@ -55,10 +54,16 @@
     min-height: 1em;
     position: relative;
     box-sizing: border-box;
-    padding: $base-margin / 5;
+    padding: $base-margin / 1.5;
+  }
+
+  .osjs-panel-item--icon {
```

Comment:
Since you're using BEN, this could be changed to:

```
.osjs-panel-item {
  &--icon {
    /* styles here */
  }
}
```

Reply:
It's funny, I used to do that often but grew to dislike it because it makes the class less searchable. You can't find in project for `osjs-panel-item--icon` and get a result anymore. I remember reading an article or tweet on that recently and I felt vindicated - I wish I could find it again.

Coding is like 90% opinions though, of which I don't really have any strong ones. So at the end of the day it's whatever you prefer. I'm flexible!

Reply:
Yeah, it's hard -- most of this stuff is so subjective. As for me I find my example more readable, but when there's not much going on it doesn't really matter. I can live with this :laughing:

Reply:
No matter how this was written, the output would be the same anyhow :sweat_smile:

Reply:
It just occurred to me I should point out that this class isn't actually applied to the same element as `osjs-panel-item`. The click event isn't being hoisted up to the containing element and I wanted to apply the styles to the exact element that has the click event. I tried the sass nesting method and it doesn't work because of that lol.

Reply:
Ah, OK :)